### PR TITLE
1.24.0 release note updates

### PR DESCRIPTION
**Pull request by mikemorris**

### CODE REVIEW

**Review by kfaseela — APPROVED** [review]

File: content/en/news/releases/1.24.x/announcing-1.24/_index.md [hunk 1]
```diff
@@ -2,7 +2,7 @@
 title: Announcing Istio 1.24.0
 linktitle: 1.24.0
 subtitle: Major Release
-description: Istio 1.24 Release Announcement.
+description: Istio 1.24 Release Announcement
```
Comment (howardjohn) [inline content/en/news/releases/1.24.x/announcing-1.24/_index.md:5]: All the others have a period, not sure why

**Review by howardjohn — APPROVED** [review]

File: content/en/news/releases/1.24.x/announcing-1.24/_index.md (hunk 1 above)
Comment (craigbox) [reply content/en/news/releases/1.24.x/announcing-1.24/_index.md:5]: I think it's a lint rule

Comment (craigbox) [reply content/en/news/releases/1.24.x/announcing-1.24/_index.md:5]: Yes, that made the build fail

> 10:44:57 AM: ERROR Page description doesn't end with a period: 'Istio 1.24 Release Announcement'

File: content/en/news/releases/1.24.x/announcing-1.24/change-notes/index.md [hunk 2]
```diff
@@ -10,10 +10,6 @@ aliases:
     - /news/announcing-1.24.0
 ---
 
-{{< warning >}}
-This is an automatically generated rough draft of the release notes and has not yet been reviewed.
-{{< /warning >}}
-
 ## Ambient
```
Comment (craigbox) [inline content/en/news/releases/1.24.x/announcing-1.24/change-notes/index.md:13]:
```suggestion
## Ambient mode
```

**Review by craigbox — APPROVED** [review]

**Pull request merged**